style.css

```diff
@@ -0,0 +1,63 @@
+* {
+    box-sizing: border-box;
+}
+
+body,
+.toolbar,
+main {
+    display: flex;
+}
+
+.toolbar,
+main {
+    flex-direction: column;
+}
+
+body  {
+    background-color: #A45EE5;
+}
+
+.toolbar {
+    background-color: #0000FF;
+    width: 150px;
+    padding: 10vw 0;
+    height: 100%;
```

Comment:
I'd suggest setting this as 100vh instead, on my screen the toolbar doesn't quite reach the bottom.